### PR TITLE
Update bazel from current master state

### PR DESCRIPTION
**Pull request by pmorie**
**What this PR does / why we need it**:

Runs update-bazel.sh on current master and commits the diffs.  Why there are diffs is not clear at all.

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

None

**Special notes for your reviewer**:

How did master get into a state where an update script yields diffs?

**Release note**:

```release-note
NONE
```

### CODE REVIEW

**Comment by spxtr** [comment]
When I run `./hack/update-bazel.sh` on the current master (54ff40670269fbae8594ec7cab4d9d3ce1b1dcd3) I don't get any diff.

**Comment by brendandburns** [comment]
Bazel build failed...

**Comment by brendandburns** [comment]
/lgtm

LGTM from the bulk LGTM tool

**Comment by brendandburns** [comment]
(sorry for the noise, I'm testing my bulk-lgtm'r)

**Comment by brendandburns** [comment]
/lgtm

LGTM from the bulk LGTM tool

**Comment by brendandburns** [comment]
/lgtm

LGTM from the bulk LGTM tool

**Comment by brendandburns** [comment]
/lgtm

LGTM from the bulk LGTM tool

**Comment by brendandburns** [comment]
(testing done)

**Comment by thockin** [comment]
/lgtm

LGTM from the bulk LGTM tool

File: examples/guestbook-go/BUILD [hunk 1]
```diff
@@ -34,6 +34,9 @@ filegroup(
 
 filegroup(
     name = "all-srcs",
-    srcs = [":package-srcs"],
+    srcs = [
+        ":package-srcs",
+        "//examples/guestbook-go/_src:all-srcs",
```
Comment (mikedanese) [inline examples/guestbook-go/BUILD]: This directory doesn't exist on master. What creates it on your dirty workspace?

**Pull request closed without merging**